jdi-light-material-ui-tests/src/main/java/io/github/com/pages/layouts/ContainerFrame.java

```diff
@@ -1,4 +1,4 @@
-package io.github.com.pages.layout;
+package io.github.com.pages.layouts;
```

Comment:
Please use "layout" variant for the folders, as its in material ui site and our storybook and site

Reply:
done